doc/nrf/device_guides/fem/fem_output_power.rst

```diff
@@ -0,0 +1,20 @@
+.. _ug_radio_fem_sw_support_mpsl_fem_output:
+
+Setting the FEM output power
+############################
+
+The ``tx_gain_db`` property in devicetree provides the FEM gain value to use with the simple GPIO FEM implementation.
+The property must represent the real gain of the FEM.
+This implementation does not support controlling the gain value during runtime.
```

Comment:
This part applies only to the simple gpio implementation. It does not apply to the nrf21540.
I would enclose both parts in sections with appropriate headers to separate between content related to Simple GPIO and nRF21540
This would have the benefit that if we add a support for yet another FEM we will add another section.

Reply:
I made a slightly different split from what you suggested.